cfpb/consumerfinance.gov
# Mega Menu: Fix issue where menu did not show on IE10 and below

In the no-js state the mega menu should override the `u-hidden` class and show the menu. The current state works when turning off JS in a modern browser, but does not work in legacy IE. 

## Changes

- Add `!important` to mega menu no-js override rules.


## How to test this PR

1. Test the PR in Sauce Labs IE10 or IE9 and see there's a menu. Compare to production and see that there is no menu.


## Screenshots

Before
<img width="1035" alt="Screen Shot 2021-07-20 at 3 37 58 PM" src="https://user-images.githubusercontent.com/704760/126385148-973ab7f1-4f89-4b59-9dd9-e09cb99d574b.png">

After (hovering over menu item)
<img width="1034" alt="Screen Shot 2021-07-20 at 3 33 42 PM" src="https://user-images.githubusercontent.com/704760/126385057-700f9362-2c0d-43a7-8664-d9832962409f.png">
